core/src/test/java/org/elasticsearch/index/query/RangeQueryBuilderTest.java

```diff
@@ -82,6 +92,13 @@ protected RangeQueryBuilder doCreateTestQueryBuilder() {
         return query;
     }
 
+    public RangeQueryBuilder createRandomStringRangeQuery() {
```

Comment:
would it make sense to move it to RandomQueryBuilder then? I don't like that class to get acces to test classes...it should rather be a utility that the test classes call...one way only.